src/webui/www/private/download.html

```diff
@@ -191,6 +192,8 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
             if (submitted)
                 window.parent.qBittorrent.Client.closeWindows();
         });
+
+        window.parent.qBittorrent.pathAutofill.attachPathAutofill();
```

Comment:
This doesn't work 🙁

Reply:
@Chocobo1  
Do you have any idea what is going on here? 

![obraz](https://github.com/qbittorrent/qBittorrent/assets/45544416/7b38fe82-72b8-47a4-b069-e87428f1d868)

I can manually get these elements, but the same method inside of the function does not work.

Reply:
I don't know why, but method `document.getElementsByClassName("pathDirectory");`

Returns anything only after investigating website code using inspector: 

![obraz](https://github.com/qbittorrent/qBittorrent/assets/45544416/f144f995-3a4a-41ee-84d0-720422384a0d)

![obraz](https://github.com/qbittorrent/qBittorrent/assets/45544416/2eac69db-e934-4e5b-ac2c-da7977bb1e3f)

Reply:
Maybe iframe is a source of problem?

Reply:
```suggestion
        window.qBittorrent.pathAutofill.attachPathAutofill();
```

Reply:
It helped. Thanks